src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => (
+
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+    {age !== undefined
+      ? <p className="Person__age">{`I am ${age}`}</p>
+      : ''}
+    <p className="Person__partner">
```

Comment:
```suggestion
    </h2>
    
    {age && (
      <p className="Person__age">
        {`I am ${age}`}
      </p>
     )}
      
    <p className="Person__partner">
```
Pay attention to the formatting. Also, no need to use ternary operator here as you only show the age or not